extensions/telegram/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "telegram",
+  "title": "Telegram Rayclient",
+  "description": "Send quick messages to your telegram contacts from raycast",
+  "icon": "telegram.png",
+  "author": "psychic_dev",
+  "categories": [
+    "Communication",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "quickmsg",
+      "title": "Quick Message",
+      "description": "send quick messages to your telegram contacts",
+      "arguments": [
+        {
+          "name": "contact",
+          "type": "text",
+          "placeholder": "contact name",
+          "required": true
+        },
+        {
+          "name": "message",
+          "type": "text",
+          "placeholder": "message",
+          "required": true
+        }
+      ],
+      "mode": "view",
+      "subtitle": "telegram",
+      "keywords": [
+        "telegram",
+        "quick",
+        "msg",
+        "message"
+      ]
+    }
+  ],
+  "preferences": [
+    {
+      "name": "api_id",
+      "title": "API ID",
+      "description": "Your telegram bot api Id",
```

Comment:
Maybe add placeholder which shows what a key looks like, maybe also how to get it?

Reply:
Sure will add